themes/default/layouts/partials/registry/package/right-nav.html

```diff
@@ -21,8 +21,18 @@ <h4 class="no-anchor">
                 </div>
             {{ end }}
         </div>
+        <div class="pl-3 flex-1 overflow-y-auto border-b-2 pb-3 pt-2">
```

Comment:
Slightly worried that putting this after the contents section will push it off the screen on many displays.  Given it's a fixed height, might it make sense to place above Contents?

Reply:
im pretty worried about putting this above contents; we are trying to keep that in a consistent place across all doc types. does it being added to the practitioner nav help at all? also have prs opened for those.

let me chat with @anita-trimbur she might have some better ideas about placement.